google-cloud-bigquery-data_transfer/docs/toc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "guides": [],
+  "services": [
+    {
+      "title": "Bigquery DataTransfer",
+      "type": "google/cloud/bigquery/data_transfer",
+      "patterns": ["\/data_transfer", "google\/api", "google\/protobuf"],
+      "nav": [
+        {
+          "title": "V1",
+          "type": "google/cloud/bigquery/data_transfer/v1",
+          "patterns": ["\/bigquery\/data_transfer\/v1", "google\/api", "google\/protobuf"],
```

Comment:
@landrito These `type` values are consumed by the Angular site app and should not have underscores:  `datatransfer`, not `data_transfer`.

Reply:
Done. Just want to make sure this correct though since this is a pattern not a type.